assets/src/edit-story/app/media/categories/actions.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import * as types from './types';
+
+export const fetchCategoriesStart = (dispatch) => ({ provider } = {}) => {
```

Comment:
Can we get rid of the default ?

Reply:
Will be investigated in #3317

Reply:
I think this can be removed, as 'provider' is a mandatory attribute.

Reply:
Unfortunately not - at least I don't get the tests to work in that case. You get the same error in https://github.com/google/web-stories-wp/pull/3318